File: automated_test_1/autotests/Helpers/webdriver_factory.py
from selenium import webdriver
from selenium.webdriver.support.ui import WebDriverWait

class WebDriverFactory:
    @staticmethod
    def get_driver():
        driver = webdriver.Chrome()
        return driver
    
    @staticmethod
    def get_wait(driver, timeout=15):
        return WebDriverWait(driver, timeout)

File: automated_test_2/autotests/Helpers/WebDriverFactory.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Support.UI;

namespace autotests.Helpers
{
    static class WebDriverFactory
    {
        public static IWebDriver CreateChromeDriver()
        {
            var driver = new ChromeDriver();
            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
            return driver;
        }

        public static WebDriverWait CreateWait(IWebDriver driver, int timeoutInSeconds = 15)
        {
            return new WebDriverWait(driver, TimeSpan.FromSeconds(timeoutInSeconds));
        }
    }
}